backfill/2022-08-31-decoder-failure/03-bigquery-etl-copy-deduplicate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -exo pipefail
+
+# This script assumes it's being run from the bigquery-etl repo, and that 02-gcp-ingestion-beam-decoder.sh
+# was run and the associated Dataflow job completed successfully.
+
+PROJECT="moz-fx-data-backfill-10"
+# FIXME update this list, see README
+TABLES=(
+)
+
+DATES=(
+    2022-08-31
+)
+
+./script/copy_deduplicate \
```

Comment:
We removed the script and moved this into the CLI tooling:

```suggestion
./bqetl copy_deduplicate \
```

Reply:
Can you add a commit with the final script you used? That will include this change

Reply:
I took a stab at updating the script. It's probably not what was run exactly but it probably isn't too important.